### DISCUSSION TITLE
Certificate download issue after enabling Sunbird-RC

### DISCUSSION THREAD

**Discussion by sravani-64** [Sunbird-Ed/Community#640] [Installation] [answered]
Hi team, 
We have enabled RC setup on sunbird 4.10 and completed all the manual configurations using https://lern.sunbird.org/use/release-notes/release-v-4.8.0-upcoming
But the user is not able to download certificate after course completion

We have connected with sunbird RC team to resolve the issue and they have identified that schema configuration and credential_template.json file are missing and redirected us to raise a ticket here again. Can someone please let us know how to add the schema configuration and credential_template.json file and complete the RC setup. Thank you

cc @Krishnaj20 @gandham-santhosh @coolbung @Renuka-S @komalm @Nithin-ns24

**Comment by gandham-santhosh** [comment]
@sravani-64  did you run this job Deploy/Sunbird-RC/UploadRCSchema ? This job should upload the schema files to the storage account

**Comment by sravani-64** [comment]
@gandham-santhosh, yes the job was triggered but it has uploaded the files to different folder in storage account. After uploading the files to required folder, we are able to download certificate. But the certificate has no username, coursename and issued date. It is printing variables 
![image](https://github.com/Sunbird-Ed/Community/assets/133327173/be595cb5-9e7a-449e-b109-27164428035e)

Can you please suggest, if it has to be checked from the RC code or backend services??

**Reply by gandham-santhosh** [reply]
Looks certificate template is incorrect. Please check the template and correct the same

**Comment by ashokreddy1208** [comment, marked as answer]
@sravani-64 This is old certificate template, new certificate template should look something like below.

<img width="1033" alt="image" src="https://github.com/Sunbird-Ed/Community/assets/58033030/b128f41a-a2af-4684-9914-31bada7778d3">

Please find new certificate template in SVG format for your reference - [Link](https://drive.google.com/file/d/1O3RF9AuQ7izAbBwxYehQ0259ORe5l-4V/view?usp=sharing)

cc @Krishnaj20

**Comment by ashokreddy1208** [comment]
@sravani-64 any update on this issue? If the issue is resolved then can you mark this thread as answered
